tests/TestRvsdgs.hpp

```diff
@@ -30,6 +28,13 @@ class RvsdgTest {
 		return module().Rvsdg();
 	}
 
+  void
+  EnsureInitialized()
```

Comment:
Could we call this function `Initialize()` (or something else) and put some documentation above it that it always needs to be called to setup the test.